libs/cardano-ledger-core/cddl/Cardano/Ledger/CDDL.hs

```diff
@@ -0,0 +1,41 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE KindSignatures #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE OverloadedLists #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE UndecidableSuperClasses #-}
+
+module Cardano.Ledger.CDDL (
```

Comment:
Let's call this module like this, in order to differentiate it from `CDDL` modules that define instances
```suggestion
module Cardano.Ledger.HasCDDL (
```